app/lib/actions/katello/repository/filtered_index_content.rb

```diff
@@ -14,13 +14,13 @@ def run
           repo = ::Katello::Repository.find(input[:id])
           unit_ids = search_units(repo)
           if repo.puppet?
-            repo.index_db_puppet_modules
+            ::Katello::PuppetModule.import_for_repository(repo)
           elsif repo.docker?
```

Comment:
Looks like this Action is called only when uploading content like rpms/puppet modules/files. We dont do this for docker manifests.  Lets remove it,